src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/NationalityValidator.java

```diff
@@ -0,0 +1,284 @@
+        modifiableListOfNationalities.add("Solomon Islander");
+        modifiableListOfNationalities.add("Somali");
+        modifiableListOfNationalities.add("South African");
+        modifiableListOfNationalities.add("South Korean");
+        modifiableListOfNationalities.add("South Sudanese");
+        modifiableListOfNationalities.add("Spanish");
+        modifiableListOfNationalities.add("Sri Lankan");
+        modifiableListOfNationalities.add("St Helenian");
+        modifiableListOfNationalities.add("St Lucian");
+        modifiableListOfNationalities.add("Stateless");
+        modifiableListOfNationalities.add("Sudanese");
+        modifiableListOfNationalities.add("Surinamese");
+        modifiableListOfNationalities.add("Swazi");
+        modifiableListOfNationalities.add("Swedish");
+        modifiableListOfNationalities.add("Swiss");
+        modifiableListOfNationalities.add("Syrian");
+        modifiableListOfNationalities.add("Taiwanese");
+        modifiableListOfNationalities.add("Tajik");
+        modifiableListOfNationalities.add("Tanzanian");
+        modifiableListOfNationalities.add("Thai");
+        modifiableListOfNationalities.add("Togolese");
+        modifiableListOfNationalities.add("Tongan");
+        modifiableListOfNationalities.add("Trinidadian");
+        modifiableListOfNationalities.add("Tristanian");
+        modifiableListOfNationalities.add("Tunisian");
+        modifiableListOfNationalities.add("Turkish");
+        modifiableListOfNationalities.add("Turkmen");
+        modifiableListOfNationalities.add("Turks and Caicos Islander");
+        modifiableListOfNationalities.add("Tuvaluan");
+        modifiableListOfNationalities.add("Ugandan");
+        modifiableListOfNationalities.add("Ukrainian");
+        modifiableListOfNationalities.add("Uruguayan");
+        modifiableListOfNationalities.add("Uzbek");
+        modifiableListOfNationalities.add("Vatican citizen");
+        modifiableListOfNationalities.add("Venezuelan");
+        modifiableListOfNationalities.add("Vietnamese");
+        modifiableListOfNationalities.add("Vincentian");
+        modifiableListOfNationalities.add("Wallisian");
+        modifiableListOfNationalities.add("Welsh");
+        modifiableListOfNationalities.add("Yemeni");
+        modifiableListOfNationalities.add("Zambian");
+        modifiableListOfNationalities.add("Zimbabwean");
+        nationalities = Collections.unmodifiableList(modifiableListOfNationalities);
+    }
+
+    private final DataSanitisation dataSanitisation;
+
+    @Autowired
+    public NationalityValidator(DataSanitisation dataSanitisation) {
+        this.dataSanitisation = dataSanitisation;
+    }
+
+    public void validateAgainstNationalityList(String qualifiedFieldName, String nationality, Errors errors, String loggingContext) {
+
+        boolean nationalityNotBlank = StringValidators.isNotBlank(nationality, qualifiedFieldName, errors, loggingContext);
+
+        if (nationalityNotBlank) {
```

Comment:
i would return from method if nationalityIsBlank rather than wrap the second if with the first one. It will reduce cognitive complexity which sonar will blocks us for if it gets too high.

Reply:
Updated.